source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/index.tsx

```diff
@@ -157,7 +157,9 @@ export default compose(
   })),
   withRole(ROLE.OrganizationAdmin, {
     redirectTo: '/',
-    checkOrganizationOf: (props: IProps) => props.organization
+    forOrganization: (props: IProps) => {
```

Comment:
this is all that was needed to fix bug https://developertown.visualstudio.com/SIL%20International%20Scriptoria/_workitems/edit/30401

Reply:
interesting...

Reply:
yeah, an organization doesn't belong to an organization, so that's where the id of undefined error was coming from